Makefile

```diff
@@ -1,24 +1,17 @@
-ANSIBLE_INSTALL_VERSION ?= 2.5.9
+ANSIBLE_INSTALL_VERSION ?= 2.6.7
```

Comment:
Do we definitely need these changes to the Makefile? It's fine if so but we will need to push these updates to the other roles at some point.

Reply:
if the changes are ok, I will send PR to scaffold repo